app-arch/brotli/brotli-1.0.2.recipe

```diff
@@ -0,0 +1,143 @@
+SUMMARY="Generic-purpose lossless compression algorithm"
+DESCRIPTION="A generic-purpose lossless compression algorithm that \
+compresses data using a combination of a modern variant of the LZ77 algorithm, \
+Huffman coding and 2nd order context modeling, with a compression ratio \
+comparable to the best currently available general-purpose compression methods. \
+It is similar in speed with deflate but offers more dense compression"
```

Comment:
I did not test this recipe but I'm quite sure there are no errors.
It looks like the only way you can improve it is by adding a full stop at the end of DESCRIPTION :)
Hmm, for SUMMARY maybe you could change "Generic-purpose" to "A generic-purpose".
BTW, for SUMMARY that's OK with no full stop.